.github/workflows/python-publish.yml

```diff
@@ -0,0 +1,31 @@
+# This workflows will upload a Python Package using Twine when a release is created
+# For more information see: https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries
```

Comment:
might steal this for DIRECT!

Reply:
It's one of the stock ones! Glad we can finally start using it